spec/features/logging/choosing_how_to_filter_spec.rb

```diff
@@ -5,8 +5,39 @@
   end
 
   context 'with no filter chosen' do
-    before { click_on "Go to search" }
+    before { click_on 'Go to search' }
 
     it_behaves_like 'errors in form'
   end
+
+  # this'll add setup to the other tests too, if implemented
+  xcontext 'with no IPs' do
```

Comment:
Presented for discussion. 

Not sure if we need to worry about this case right now, got a few ways we could handle it, too (my other major idea was "hide logs in the subnavigation menu 'til you've got an IP")

Reply:
Hiding it seems like a better method. The need to view logs doesn't seem realistic without an IP to search against and username logs shouldn't exist if there are no IPs to receive the traffic.

Reply:
With some time to think about it, I agree. It's a pretty simple solution, too.